starter-code/css/main.css

```diff
@@ -6,3 +6,112 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  font-family: Helvetica,Arial,sans-serif;
+  font-weight: lighter;
+  margin: 0;
+}
+
+/*HEADER SECTION*/
+
+header {
+  padding: 10px 40px 0px 40px;
+}
+
+header img {
+  width: 14%;
+}
+
+ul {
```

Comment:
A more specific selector would be more appropriate. You don't want those rules to affect `every` list on the page (even though there is only one yet).